src/demo-app/tabs/tabs-demo.ts

```diff
@@ -133,3 +133,16 @@ export class RainyTabContent {}
   template: 'This is the routed body of the foggy tab.',
 })
 export class FoggyTabContent {}
+
+@Component({
+  moduleId: module.id,
+  selector: 'counter',
+  template: `<span>{{count}}</span>`
+ })
+ export class Counter {
+  count = 0;
+  ngOnInit() {
+    this.count++;
+    console.log('Counting...', this.count);
```

Comment:
Can you provide some comments about why we are `console.log`ing and what the use of this Counter is?